src/commands/TimeCommand.ts

```diff
@@ -0,0 +1,157 @@
+import { Command } from "../Command";
+import { CommandInteraction, GuildMember } from "discord.js";
+import {
+	SlashCommandBooleanOption,
+	SlashCommandBuilder,
+	SlashCommandIntegerOption,
+	SlashCommandStringOption,
+	SlashCommandSubcommandsOnlyBuilder,
+} from "@discordjs/builders";
+
+class Reminder extends Command {
+	constructor() {
+		super("reminder");
+	}
+
+	elapse(interaction: CommandInteraction, callAll: boolean, caller: string, message = ""): void {
+		if (callAll) {
+			interaction.followUp("@everyone, Time's up!");
+		} else {
+			interaction.followUp("<@!" + caller + "> Time's up!");
+		}
+		if (message != "") {
+			interaction.followUp("Message: " + message);
+		}
+		this.m_id -= 1;
+	}
+
+	private rmdDatabase: {
```

Comment:
we have propper databases already (iirc) which last longer than the current runtime as they are saved cross processes

Reply:
we do?

Reply:
db.ts exists. ain't much in it, but it does indeed exist